src/js.cookie.js

```diff
@@ -52,8 +52,12 @@
 				path: '/'
 			}, api.defaults, attributes);
 
-			if (typeof attributes.expires === 'number') {
-				attributes.expires = new Date(new Date() * 1 + attributes.expires * 864e+5);
+			// Check Date validity
+			if (attributes.expires instanceof Date) {
+				attributes.expires = isNaN(Number(attributes.expires)) ? '' : attributes.expires;
+			} else if (attributes.expires) {
+				var expires = parseFloat(attributes.expires);
+				attributes.expires = isNaN(expires) ? '' : new Date(new Date() * 1 + expires * 864e+5);
```

Comment:
Need a failing test to justify the code change, see https://github.com/js-cookie/js-cookie/blob/d2724a87a29cbf7ee539452fb42b3e3a1aff95a8/CONTRIBUTING.md#pull-requests

Reply:
I didn’t even mention lack of tests because I’m not sure we want to relax the api in the first place.

Reply:
That's right, agree on that